hail/python/hailtop/httpx.py

```diff
@@ -38,8 +38,8 @@ class ClientResponse:
     def __init__(self, client_response: aiohttp.ClientResponse):
         self.client_response = client_response
 
-    def release(self) -> None:
-        return self.client_response.release()
+    async def release(self) -> None:
```

Comment:
What is the reason for this change? I looked at `aiohttp.ClientResponse` and `release` is not async.

Reply:
release is async: https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientResponse.release

I returns "noop" which is a generator which makes the function async.